sktime/forecasting/theta_new.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file).
+"""Modular ThetaForecaster."""
+
+__author__ = ["GuzalBulatova"]
+__all__ = ["ThetaNewForecaster"]
+
+from sktime.forecasting.base._meta import _HeterogenousEnsembleForecaster
+from sktime.forecasting.compose import ColumnEnsembleForecaster
+from sktime.forecasting.compose._ensemble import _aggregate
+from sktime.forecasting.compose._pipeline import TransformedTargetForecaster
+from sktime.forecasting.exp_smoothing import ExponentialSmoothing
+from sktime.forecasting.trend import PolynomialTrendForecaster
+from sktime.transformations.series.theta import ThetaLinesTransformer
+
+
+class ThetaNewForecaster(_HeterogenousEnsembleForecaster):
```

Comment:
One small thought here - I *think* that there seems to be a movement away from using `HeterogenousEnsembleForecaster` and instead just using it's parent `HeterogenousMetaEstimator` instead!  I think you could probably do that here as well (though I am sorry if I am wrong and breaks something!). :)

Reply:
We inherit from Forecaster here because we need the get and set params methods. We do eventually overwrite them so maybe makes sense to inherit from MetaEstimator instead actually. @fkiraly what do you think?

Reply:
hm, not sure what the consequences would be of that. Arer we using any of the forecaster specific methods?